### PR TITLE
flaky test: kubectl describe serviceaccout not stable sequence output…

### LINKED ISSUE #54564: test flake: kubectl describe serviceaccount unittest will test occasionally
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line: 
>
/kind bug
/kind flake
/kind cli

**What happened**:

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration**:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by zjj2wry**
…,so cause ci faild

**What this PR does / why we need it**:
Fixes #54564

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
`Image pull secrets:  test-local-ref (not found)` and `Mountable secrets:   test-objectref (not found)` will swop sometimes，so cause ci faild sometimes
**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by zjj2wry** [comment]
Fixes #54564

File: pkg/printers/internalversion/describe_test.go [hunk 1]
```diff
@@ -1682,18 +1682,12 @@ func TestDescribeServiceAccount(t *testing.T) {
...
-Labels:              <none>
-Annotations:         <none>
-Image pull secrets:  test-local-ref (not found)
-Mountable secrets:   test-objectref (not found)
-Tokens:              <none>
-Events:              <none>` + "\n"
-	if out != expectedOut {
-		t.Errorf("expected : %q\n but got output:\n %q", expectedOut, out)
+	expectedOut := []string{"bar", "foo", "Image pull secrets", "test-local-ref (not found)", "Mountable secrets:", "test-objectref (not found)"}
+	for _, expected := range expectedOut {
```
Comment (dixudx) [inline pkg/printers/internalversion/describe_test.go]: Well, I don't like changing to this, since `expectedOut` is exactly what we expect to see.

**Review by dixudx — CHANGES_REQUESTED** [review]

**Review by ericchiang — COMMENTED** [review]
Since the flake is that the lines can be out of order why not just sort the lines?

```diff
+       sortLines := func(s string) string {
+               lines := strings.Split(out, "\n")
+               sort.Strings(lines)
+               return strings.Join(lines, "\n")
+       }
+
+       out = sortLines(out)
+       expectedOut = sortLines(expectedOut)
```

**Comment by deads2k** [comment]
I don't think this is the correct solution.  The describe itself should be predictable. See https://github.com/kubernetes/kubernetes/pull/54628

**Comment by zjj2wry** [comment]
@deads2k got, will respect it in future, thank you ~

**Pull request closed without merging**